net.technicpack.TechnicLauncher.yml

```
@@ -0,0 +1,60 @@
+app-id: net.technicpack.TechnicLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk8
+command: run.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
+    - --socket=pulseaudio
+    - --share=network
+    - --persist=.technic
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk8/install.sh
+
+    - name: techniclauncher
+      buildsystem: simple
+      build-commands:
+          - install -D TechnicLauncher.jar /app/bin/TechnicLauncher.jar
+      sources:
+          - type: file
+            path: TechnicLauncher.jar
+            url: https://launcher.technicpack.net/launcher4/709/TechnicLauncher.jar
+            sha256: b8b9dab63a848c219906ea76258987d98cf89df3bffc672bec2f9d491e5b8cf3
+            
+    - name: logo
+      buildsystem: simple
+      build-commands:
+          - install -D icon.svg /app/share/icons/hicolor/512x512/apps/net.technicpack.TechnicLauncher.svg
```

```suggestion
          - install -D icon.svg /app/share/icons/hicolor/scalable/apps/net.technicpack.TechnicLauncher.svg
```
I'm actually surprised this didn't fail.

---

I didn't know about this at all